tests/integration/goerly/trace_call/test_1.json

```diff
@@ -1,4773 +0,0 @@
-[
```

Comment:
May I ask why are we deleting this integration test?

Reply:
As the standard (by @lupin012) is to use a double digit to enumerate test (from 00 to 99), the test_01.json is already present (maybe it was a refuse..)